doc/doc-support/lib-function-docs.nix

```diff
@@ -1,12 +1,10 @@
-# Generates the documentation for library functons via nixdoc. To add
-# another library function file to this list, the include list in the
-# file `doc/functions/library.xml` must also be updated.
```

Comment:
Isn't this still the case?
I think it's quite feasible to generate that file though, in the derivation below.

Reply:
I moved the comment along with the list, in `doc/doc-support/default.nix`. But yes that would be nice.